SparkPost/support-docs
# WE-378: prefetching error on page load and hover

### What Changed

Updates the netlify-nextjs plugin to latest, fixes a content issue where links were incorrect.

*NOTE: Technically, this is fixed in WE-377 because there was a newer version that came out (4.1.1) that WE-377 updated to. But there are changes in this PR we should still push through.

### How To Test or Verify
- Load the home page and verify the 404 console errors are not appearing on page load or link hover.
  - https://deploy-preview-639--support-docs.netlify.app/docs
